openpype/settings/defaults/project_settings/maya.json

```diff
@@ -789,129 +789,134 @@
+                            "hwFogFalloff": "0",
+                            "hwFogDensity": 0.0,
+                            "hwFogStart": 0,
+                            "hwFogEnd": 100,
+                            "hwFogAlpha": 0,
+                            "hwFogColorR": 1.0,
+                            "hwFogColorG": 1.0,
+                            "hwFogColorB": 1.0,
+                            "motionBlurEnable": false,
+                            "motionBlurSampleCount": 0,
+                            "motionBlurShutterOpenFraction": 0.2,
+                            "cameras": false,
+                            "clipGhosts": false,
+                            "deformers": false,
+                            "dimensions": false,
+                            "dynamicConstraints": false,
+                            "dynamics": false,
+                            "fluids": false,
+                            "follicles": false,
+                            "gpuCacheDisplayFilter": false,
+                            "greasePencils": false,
+                            "grid": false,
+                            "hairSystems": true,
+                            "handles": false,
+                            "headsUpDisplay": false,
+                            "ikHandles": false,
+                            "imagePlane": true,
+                            "joints": false,
+                            "lights": false,
+                            "locators": false,
+                            "manipulators": false,
+                            "motionTrails": false,
+                            "nCloths": false,
+                            "nParticles": false,
+                            "nRigids": false,
+                            "controlVertices": false,
+                            "nurbsCurves": false,
+                            "hulls": false,
+                            "nurbsSurfaces": false,
+                            "particleInstancers": false,
+                            "pivots": false,
+                            "planes": false,
+                            "pluginShapes": false,
+                            "polymeshes": true,
+                            "strokes": false,
+                            "subdivSurfaces": false,
+                            "textures": false
+                        },
+                        "Camera Options": {
+                            "displayGateMask": false,
+                            "displayResolution": false,
+                            "displayFilmGate": false,
+                            "displayFieldChart": false,
+                            "displaySafeAction": false,
+                            "displaySafeTitle": false,
+                            "displayFilmPivot": false,
+                            "displayFilmOrigin": false,
+                            "overscan": 1.0
+                        }
                     }
```

Comment:
It seems you've resolved a merge conflict wrong? You're missing the `pluginObjects` entry and have the `gpuCacheDisplayFilter` back in the hierarchy above it.

Reply:
Sorry, yup bad conflict merging by me.